web/components/contract/contract-card.tsx

```diff
@@ -89,72 +91,42 @@ export function ContractCard(props: {
         className
       )}
     >
-      <Col className="relative flex-1 gap-3 py-4 pb-12  pl-6">
+      <Col className="relative flex-1 gap-3 py-4 pb-12 ">
         <AvatarDetails
           contract={contract}
-          className={'hidden md:inline-flex'}
+          className={'hidden pl-2 md:inline-flex'}
           noLink={noLinkAvatar}
         />
-        <div
-          className={clsx(
-            'break-anywhere font-semibold text-indigo-700 group-hover:underline group-hover:decoration-indigo-400 group-hover:decoration-2',
-            questionClass
+        {showImage && (
+          <img
+            className="h-60 w-full object-cover "
+            src={
+              'https://firebasestorage.googleapis.com/v0/b/mantic-markets.appspot.com/o/dream%2FykU5KOeiSQ.png?alt=media&token=f7c69a0f-8f71-4758-996c-0b6411d16875'
```

Comment:
Will change this to contract.coverImgUrl before shipping - leaving for now so you can try the dev builds.